### DISCUSSION TITLE
Get messages which have been sent by the bot itself

### DISCUSSION THREAD

**Discussion by fkuruh** [python-telegram-bot/python-telegram-bot#3198] [Q&A]
Hi there, im currently working on some SMS TAN automation.
I have set up an application on my android phone which forwards every incomming sms to my telegram bot. The forwarded messages are sent by my bot and i need to read them out.
I tried using getUpdates but i only can see messages which have been sent manually by myself.
Is there any possibility to read out EVERY single message (even sent by the bot itself)?

Thanks in advance for helping me ! :)

**Comment by Poolitzer** [comment]
No, you don't get update from messages send by yourself.

A very hacky workaround could be to create a channel, attach a discussion group, and send the message into the channel.

But you should really redesign this whole process, this is so error prone. Just build a small rest api and send it to the backend of the bot that way.
